### PR TITLE
Add spec for bake_chapter_title

### PR DESCRIPTION
**Pull request by kswanson33**
#47 
Copies implementation for BakeChapterTitle from [recipes](https://github.com/openstax/recipes/blob/main/books/chemistry2e/bake#L39-L49).

### CODE REVIEW

File: spec/.rubocop.yml [hunk 1]
```diff
@@ -23,3 +23,6 @@ Style/SymbolProc:
 
 RSpec/FilePath:
   Enabled: false
+
+RSpec/ExampleLength:
+  Enabled: false
```
Comment (kswanson33) [inline spec/.rubocop.yml:28]: This addition gets rid of the 5-line maximum for a spec example -- lmk if we want to do something different.

Comment (jpslav) [reply spec/.rubocop.yml:28]: Yeah I think this is fine.  in non rspec-land, there are some settings like

```ruby
Metrics/MethodLength:
  Max: 50
  CountAsOne: ['heredoc']
```

that would count your big heredoc string as one line, but I don't think the rspec cop has this feature

File: lib/kitchen/directions/bake_chapter_title/main.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module Kitchen
+  module Directions
+    module BakeChapterTitle
+      def self.v1(book:)
+        V1.new.bake(book: book)
```
Comment (jpslav) [inline lib/kitchen/directions/bake_chapter_title/main.rb:5]: You can write a simple test for this line:

```ruby
RSpec.describe Kitchen::Directions::BakeChapterTitle do
  it 'calls v1' do
    expect_any_instance_of(Kitchen::Directions::BakeChapterTitle::V1).to receive(:bake).with(book: "blah")
    described_class.v1(book: "blah")
  end
end
```

Later in this file we'd add `it 'calls v2'` etc

File: lib/kitchen/directions/bake_chapter_title/v1.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+module Kitchen::Directions::BakeChapterTitle
+  class V1
+    def bake(book:)
+      book.chapters.each do |chapter|
+        fix_up_chapter_title(chapter: chapter)
+      end
+    end
+
+    def fix_up_chapter_title(chapter:)
+      # Fix up chapter titles
```
Comment (jpslav) [inline lib/kitchen/directions/bake_chapter_title/v1.rb:10]: can probably ditch the comment as the method name tells us what we need to know

File: lib/kitchen/directions/bake_chapter_title/main.rb (hunk 2 above)
Comment (jpslav) [reply lib/kitchen/directions/bake_chapter_title/main.rb:5]: I'm sure this is missing for the other existing directions specs too

**Review by jpslav — CHANGES_REQUESTED** [review]

**Review by jpslav — APPROVED** [review]

**Pull request merged**